### DISCUSSION TITLE
Change preset to eco / comfort instead of turning climate to on / off

### DISCUSSION THREAD

**Discussion by Julien-Decoen** [jseidl/magic-areas#262] [Ideas / Feature Requests]
Hi ! 

I'm only turning my climate off when it's the summer. 
So, It could be awesome to changing the preset of my climate to ECO when the area become clear, and change the preset to comfort, when the area become to the selected state.

Thanks,
Lou_Juicy

**Comment by jseidl** [comment]
This actually needs to be an option on the climate group setting because we can't infer automatically what you want, but good call!

**Reply by Nuuki9** [reply]
Was there any progress on this? Its not typical to want to turn heating off entirely in an unoccipoed room, and the ability to define climate presets to trigger when an area is occupied or unoccupied, versus simply turning it on or off, would be great.

**Reply by tbrasser** [reply]
I've been using Versatile Thermostat in the meantime for this functionality.

**Comment by jseidl** [comment]
Climate Groups (among other features) is still very raw and basic. When I first created it, it was mostly to control [generic thermostats](https://www.home-assistant.io/integrations/generic_thermostat/) which were controlling fans. It really has room for improvement and this idea of having ability to specify presets per room state would be great.

The way I manage the feature request queue is looking at upvotes as a gauge of what feature people want more and this one has only one upvote apart from 3 people interested, so it's hard to me to pick up an "1 upvote" FR vs a "3 upvote" FR and so on.

PLEASE UPVOTE! (the original post, on the top) I just did because I find this FR cool too

**Comment by jseidl** [comment]
Good news folks. Climate groups is being deprecated in favor of "Climate control" 

PR is done https://github.com/jseidl/hass-magic_areas/pull/493 being merged this week. Allows you to map presets to area states on a given thermostat. I've been using for both my upstairs and downstairs thermostats on the meta areas for these floors.
